nemo_skills/evaluation/evaluator/nvembed_judge.py

```
@@ -42,7 +42,7 @@ def install_packages():
     """Install required packages for NVEmbed evaluation."""
     LOG.info("Installing required packages...")
     subprocess.run(
-        ["pip", "install", "-q", "datasets", "einops", "transformers==4.42.4", "tqdm"],
+        ["pip", "install", "-q", "numpy<2", "datasets", "einops", "transformers==4.42.4", "tqdm"],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)